main.js

```diff
@@ -0,0 +1,149 @@
+//Unit 1
+const multiplicationTable = (x) => {
+  for (let i = 1; i <= 10; i++)
+  console.log(x + 'x' + i + '=' + (x * i))
+}
+
+//Unit 2
+function printEvenNumbers(n) {
+  for (let i = 1; i <= n; i++) {
+    if (i % 2 == 0) {
```

Comment:
```suggestion
    if (i % 2 === 0) {
```
dùng `===` thay cho `==`